graphql/resolvers.go

```diff
@@ -870,6 +870,25 @@ func (r *queryResolver) SiteBanner(ctx context.Context) (*restModel.APIBanner, e
 	return &banner, nil
 }
 
+func (r *queryResolver) Host(ctx context.Context, limit *int, page *int, hostID *string) (*restModel.APIHost, error) {
```

Comment:
you removed `limit` and `page` from the query params, which was the right move, now you need to update the resolver signature to match.